auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/RestWrapper.java

```diff
@@ -54,6 +45,15 @@ public RestWrapper(Object objectToSerialize, String result) {
         }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)